tests/integrationv2/providers.py

```diff
@@ -11,6 +14,22 @@
 }
 
 
+def get_expected_s2n_version(protocol, provider):
```

Comment:
Why did you move this here from utils? That's what's triggering a lot of the changes in this PR. You might want to either not do it or do it in a different PR to slim this one down. Uninteresting changes like this tend to overwhelm / hide the interesting changes.

Reply:
Reverted.